jf_agent/main.py

```diff
@@ -211,7 +210,7 @@ class BadConfigException(Exception):
 
 JellyfishEndpointInfo = namedtuple(
     'JellyfishEndpointInfo',
-    ['s3_uri_prefix', 'aws_access_key_id', 'aws_secret_access_key', 'git_instance_info'],
+    ['git_instance_info'],
```

Comment:
it seems a little bizarre to have a namedtuple with only one thing in it

Reply:
Yea, I should update this - doesn't seem to serve a purpose anymore.